### PR TITLE
Upgrade pna.h and add crc32.h

### PR DESCRIPTION
**Pull request by vbnogueira**

### CODE REVIEW

File: vm/crc32.h [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    0x6f2c5505, 0xa386559b, 0x61d761c,  0xcab77682, 0x44387161, 0x889271ff,
+    0x825778e6, 0x4efd7878, 0xc0727f9b, 0xcd87f05,  0xd5f86da9, 0x19526d37,
+    0x97dd6ad4, 0x5b776a4a, 0x51b26353, 0x9d1863cd, 0x1397642e, 0xdf3d64b0,
+    0x83d02561, 0x4f7a25ff, 0xc1f5221c, 0xd5f2282,  0x79a2b9b,  0xcb302b05,
+    0x45bf2ce6, 0x89152c78, 0x50353ed4, 0x9c9f3e4a, 0x121039a9, 0xdeba3937,
+    0xd47f302e, 0x18d530b0, 0x965a3753, 0x5af037cd, 0xff6b144a, 0x33c114d4,
+    0xbd4e1337, 0x71e413a9, 0x7b211ab0, 0xb78b1a2e, 0x39041dcd, 0xf5ae1d53,
+    0x2c8e0fff, 0xe0240f61, 0x6eab0882, 0xa201081c, 0xa8c40105, 0x646e019b,
+    0xeae10678, 0x264b06e6,
+};
```
Comment (jhsmt) [inline vm/crc32.h:345]: Why is this new?

File: vm/pna.h [hunk 2]
```diff
@@ -80,101 +80,107 @@ struct clone_session_entry {
 struct p4tc_table_entry_act_bpf_params__local {
         u32 pipeid;
         u32 tblid;
-} __attribute__((preserve_access_index));
+};
 
 struct __attribute__((__packed__)) p4tc_table_entry_act_bpf {
         u32 act_id;
+	u32 hit:1,
```
Comment (jhsmt) [inline vm/pna.h:87]: Remember we saw this as 3 bits - should we be concerned on efficiency/padding etc - do we need "packed"?

**Review by jhsmt — CHANGES_REQUESTED** [review]
Just double check what i asked first

File: vm/crc32.h (hunk 1 above)
Comment (vbnogueira) [reply vm/crc32.h:345]: The previous `pna.h` file didn't have it included

File: vm/pna.h (hunk 2 above)
Comment (vbnogueira) [reply vm/pna.h:87]: This struct is already using `__attribute__((__packed__))`

**Pull request merged**